### DISCUSSION TITLE
how to install?

### DISCUSSION THREAD

**Discussion by GhostBusterRovo** [gnembon/fabric-carpet#1785] [Q&A]
how do i install it

**Comment by altrisi** [comment]
First install Fabric Loader (see https://fabricmc.net/use), then put the jar in releases (from either [GtiHub](https://github.com/gnembon/fabric-carpet/releases/), [Modrinth](https://modrinth.com/mod/carpet/versions) or [Curseforge](https://www.curseforge.com/minecraft/mc-mods/carpet)) into your mods folder.
